cli/fmt.rs

```diff
@@ -29,13 +29,19 @@ const BOM_CHAR: char = '\u{FEFF}';
 ///
 /// First argument supports globs, and if it is `None`
 /// then the current directory is recursively walked.
-pub async fn format(args: Vec<String>, check: bool) -> Result<(), ErrBox> {
+pub async fn format(
+  args: Vec<String>,
+  check: bool,
+  ignore: Vec<String>,
+) -> Result<(), ErrBox> {
   if args.len() == 1 && args[0] == "-" {
     return format_stdin(check);
   }
-
-  let target_files = collect_files(args)?;
-
+  let mut target_files = collect_files(args)?;
+  if !ignore.is_empty() {
+    let ignore_files = collect_files(ignore)?;
+    target_files.retain(|f| !ignore_files.contains(&f));
+  }
```

Comment:
Does this work for filtering out by parent paths, like your use case suggests?

Reply:
Yup. collect_files gives all the files in the dir (if a dir is passed)

Reply:
> So the filtering is only for files and not parent folders as the folders content are resolved to its files.

So `deno fmt --ignore=./node_modules/`  won't work, then? That probably needs to be fixed since ignoring by directory is a big use case.

Reply:
@nayeemrmn It works for both files and folders. By 'filtering' I mean that the folders are resolved into their files by `collect_files` method.